reservation_units/utils/reservation_unit_payment_helper.py

```diff
@@ -30,3 +30,15 @@ def requires_product_mapping_update(cls, reservation_unit) -> bool:
             return True
 
         return False
+
+    @classmethod
+    def get_accounting(cls, reservation_unit) -> Optional[PaymentAccounting]:
+        if reservation_unit.payment_accounting is not None:
+            return reservation_unit.payment_accounting
+        if (
+            reservation_unit.unit
+            and reservation_unit.unit.payment_accounting is not None
```

Comment:
Minor tweaky; I think we can check just `if reservation_unit.unit` and then return the `payment_accounting` even if `None` since we're returning `None` in the below

Reply:
True! I simplified the code 👍